packages/contracts/test/cyberDropBase.spec.ts

```diff
@@ -48,77 +46,73 @@ describe('CyberDropBase', function () {
     )
   })
 
-  describe('CreateDrop', () => {
-    it('Should create drop', async () => {
+  describe('DropMintCounter', () => {
+    it('Should return zero if drop doesnt exist', async () => {
```

Comment:
'Should return zero if drop doesnt exist' => revertedWith('DNE'), title and content are different